docs/96_faq/index.md

```diff
@@ -73,6 +73,24 @@ to update the key pairs associated with an account permission. This command repl
 
 If you are using a third party wallet with a UI, they may provide the ability to update your account public and private key pairs.
 
+### What is a digital wallet
+
+A digital wallet, also called simply "wallet", is where the keys are stored. A wallet can also allow you to:
```

Comment:
Suggestion if it makes sense:

 ".....is where the cryptographic keys are stored."